index.bs

```diff
@@ -443,51 +449,30 @@ Verify source for all of the above. See [Microsoft Typography](https://docs.micr
 
 </aside>
 
-Issue: Include `name` ID 2 (Font subfamily, e.g. "Regular") as well?
+Issue: Include `name` ID 2 (Font subfamily/style, e.g. "Regular", "Bold", etc.) as well?
 
 Issue: Include `name` ID 3 (Unique identifier) as well?
 
 
 <div class="domintro note">
 
-    : await |map| = |metadata| . {{FontMetadata/getTables()}}
-    : await |map| = |metadata| . {{FontMetadata/getTables()|getTables}}(|tableNames|)
-    :: Request the internal tables of |metadata|. The result |map| is {{FontTableMap}} with an API like a {{ECMAScript/Map}}, where the keys are matching table names and the values are {{Blob}}s with the table binary data. If |tableNames| is not specified, all tables are returned.
+    : await |blob| = |metadata| . {{FontMetadata/blob()}}
+    :: Request the font data of |metadata|. The result |blob| contains [=font representation/data bytes=].
 
 </div>
 
-<xmp class=idl>
-[Exposed=(Window,Worker)]
-interface FontTableMap {
-  readonly maplike<ByteString, Blob>;
-};
-</xmp>
-
-
 <div algorithm>
 
-The <dfn method for=FontMetadata>getTables(|tableNames|)</dfn> method steps are:
+The <dfn method for=FontMetadata>blob()</dfn> method steps are:
 
 1. Let |promise| be [=/a new promise=].
-1. Otherwise, if |tableNames| was given but is [=list/empty=], then [=/reject=] |promise| with a {{TypeError}}.
-1. Otherwise, run these steps [=in parallel=]:
-    1. Let |permission| be the result of [=requesting permission to use=] {{PermissionName/"local-fonts"}}.
+1. Run these steps [=in parallel=]:
+    1. Let |permission| be the result of [=requesting permission to use=] {{PermissionName/"font-access"}}.
     1. If |permission| is not {{PermissionState/"granted"}}, then [=/reject=] |promise| with a "{{NotAllowedError}}" {{DOMException}}, and abort these steps.
-    1. Let |map| be a new {{FontTableMap}}.
-    1. Let |backing| be |map|'s \[[BackingMap]] (an {{ECMAScript/Map}}.)
-    1. Let |font| be [=/this=]'s [=/font representation=].
-    1. [=list/For each=] |table| of |font|'s [=font representation/table list=]:
-        1. Let |tag| be |table|'s [=font table/tag=].
-         1. If |tableNames| was given and does not [=list/contain=] |tag|, then [=iteration/continue=].
-        1. Let |blob| be a new {{Blob}} whose contents are |table|'s [=font table/data bytes=] and {{Blob/type}} attribute is \``application/octet-stream`\`.
-        1. Append the [=ecma-record|Record=] { \[[Key]]: |tag|, \[[Value]]: |blob| } to |backing|.\[[MapData]].
-    1. [=/Resolve=] |promise| with |map|.
+    1. Let |blob| be a new {{Blob}} whose contents are |metadata|'s [=font representation/data bytes=] and {{Blob/type}} attribute is \``application/octet-stream`\`.
```

Comment:
I thought the explanation of [=font representation/data bytes=] as being concept/context. And the link between `Font Metadata` and `font representation` is made with this sentence:

```
Each FontMetadata has an associated font representation.
```

`font representation`, I thought was important to stand on its own because it delved into the concept of tables and specifically, the name table, to explain where `postscriptName`, `fullName`, and `family` come from.

Do you have thoughts on how this could flow better?